tests/integrations/celery/test_celery.py

```diff
@@ -313,56 +313,57 @@ def dummy_task(self):
         assert e["type"] == "ZeroDivisionError"
 
 
-@pytest.mark.forked
-def test_redis_backend_trace_propagation(init_celery, capture_events_forksafe, tmpdir):
-    celery = init_celery(traces_sample_rate=1.0, backend="redis", debug=True)
+# TODO: This test is hanging when running test with `tox --parallel auto`. Find out why and fix it
```

Comment:
can you use 
https://docs.pytest.org/en/7.1.x/reference/reference.html#pytest-mark-skip

Reply:
Done